.github/workflows/pytest.yml

```diff
@@ -9,26 +9,65 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8]
+        use-conda: [true, false]
+        use-dist: [false]
         include:
           - python-version: 3.8
             code-cov: true
+          - python-version: 3.7
+            use-conda: false
+            use-dist: true
       fail-fast:  false
-      max-parallel: 2
 
     steps:
     - uses: actions/checkout@v2
     - name: Setup Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install test dependencies
+    - name: Conda Install test dependencies 
+      if: matrix.use-conda == true
+      run: |
+        # Miniconda is available in $CONDA env var
+        $CONDA/bin/conda create -n testenv --yes pip wheel gxx_linux-64 gcc_linux-64 swig python=${{ matrix.python-version }}
+        $CONDA/envs/testenv/bin/python3 -m pip install --upgrade pip
+        $CONDA/envs/testenv/bin/pip3 install -e .[test]
+    - name: Install test dependencies 
+      if: matrix.use-conda == false && matrix.use-dist == false
       run: |
         python -m pip install --upgrade pip
         pip install -e .[test]
+        sudo apt-get update
+        sudo apt-get remove swig
+        sudo apt-get install swig3.0
+        sudo ln -s /usr/bin/swig3.0 /usr/bin/swig
+    - name: Dist Install test dependencies 
+      if: matrix.use-conda == false && matrix.use-dist == true
+      run: |
+        python -m pip install --upgrade pip
+        sudo apt-get update
+        sudo apt-get remove swig
+        sudo apt-get install swig3.0
+        sudo ln -s /usr/bin/swig3.0 /usr/bin/swig
+        python setup.py sdist
+        last_dist=$(ls -t dist/auto-sklearn-*.tar.gz | head -n 1)
+        pip install $last_dist
```

Comment:
This is now missing:
```
    twine_output=`twine check "$dist"`
    if [[ "$twine_output" != "Checking $dist: PASSED" ]]; then
        echo $twine_output
        exit 1
    else
        echo "Check with Twine: OK: $twine_output"
    fi
```

Reply:
Hi Matthias, please let me know if this is not the check you were looking for:
https://github.com/automl/auto-sklearn/blob/40e59eb0c4105ad75c17c20cac8e54dd27a935d7/.github/workflows/dist.yml#L21

Reply:
Yes, it is, but it does net check that the output value is fine.